airflow/contrib/hooks/emr_hook.py

```diff
@@ -51,11 +51,12 @@ def get_cluster_id_by_name(self, emr_cluster_name, cluster_states):
 
         if len(matching_clusters) == 1:
             cluster_id = matching_clusters[0]['Id']
-            self.log.info('Found cluster name = %s id = %s' % (emr_cluster_name, cluster_id))
+            self.log.info(f'Found cluster name = {emr_cluster_name} id = {cluster_id}')
```

Comment:
```suggestion
            self.log.info('Found cluster name = %s id = %s', emr_cluster_name, cluster_id)
```

In case of logging we do formatting this way.